atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -309,14 +309,8 @@ private void registerMetrics(GenericObjectPool<CassandraClient> pool) {
         registerPoolMetric("meanBorrowWaitTimeMillis", pool::getMeanBorrowWaitTimeMillis);
```

Comment:
meanIdleTimeMillis is interesting: I guess it might tell you if your min connection pool size is too large, but I don't think anyone ever cares? Also the idle timeout is 10 minutes, so I'd vote for canning this one too, unless you can think of a purpose for it?

Reply:
Yeah, didn't really have a purpose for this: the main one was just if it was very close to 10m that could be interesting, but agree that it's not particularly valuable. Let's kill it